pallets/nfts/src/lib.rs

```diff
@@ -1371,6 +1529,55 @@ pub mod pallet {
 			Self::do_clear_all_transfer_approvals(maybe_check_origin, collection, item)
 		}
 
+		/// Cancel all the collection approvals.
+		///
+		/// Origin must be Signed.
+		///
+		/// Arguments:
+		/// - `collection`: The collection whose approvals will be cleared.
+		/// - `witness_approvals`: Information on the collection approvals cleared. This must be
+		///   correct.
+		///
+		/// Emits `AllApprovalsCancelled` on success.
+		///
+		/// Weight: `O(1)`
+		#[pallet::call_index(22)]
+		#[pallet::weight(T::WeightInfo::clear_all_collection_approvals(*witness_approvals))]
+		pub fn clear_all_collection_approvals(
+			origin: OriginFor<T>,
+			collection: T::CollectionId,
+			witness_approvals: u32,
+		) -> DispatchResult {
+			let origin = ensure_signed(origin)?;
```

Comment:
owner

Reply:
Please see the `set_attribute` first. Other places use `origin` why this code use `owner` will create inconsistency

Reply:
So please say that in stead of resolving. You resolve a comment when you are sure the other person is aware and agrees with the conversation being resolved. Others in the team let the other person resolve which I think is a good practise as right now I have no oversight in what is happening with my comments as you resolve them all, even the ones that you do nothing with.

Reply:
Btw, I see that you changed the comments and used the `owner` but not changed the `origin` parameter. Apologies, but it is really hard to follow you like this and this is not helping to progress.

Reply:
Which one is it?